src/web/page/templates.rs

```diff
@@ -100,7 +100,7 @@ fn load_rustc_resource_suffix() -> Result<String> {
 }
 
 pub(super) fn load_templates() -> TeraResult<Tera> {
-    let mut tera = Tera::new("templates-tera/**/*")?;
+    let mut tera = Tera::new("tera-templates/**/*")?;
```

Comment:
Do we use the subdirectories?
```suggestion
    let mut tera = Tera::new("tera-templates/*")?;
```

Reply:
Yes, in later stages we do